x-pack/plugins/fleet/server/constants/fleet_es_assets.ts

```diff
@@ -36,10 +40,14 @@ export const FLEET_GLOBAL_COMPONENT_TEMPLATE_CONTENT = {
   },
 };
 
-export const FLEET_FINAL_PIPELINE_VERSION = 1;
+export const FLEET_FINAL_PIPELINE_VERSION = 2;
+
 // If the content is updated you probably need to update the FLEET_FINAL_PIPELINE_VERSION too to allow upgrade of the pipeline
 export const FLEET_FINAL_PIPELINE_CONTENT = `---
 version: ${FLEET_FINAL_PIPELINE_VERSION}
+_meta:
```

Comment:
Interpolating the full `_meta` object generated by `getESAssetMetadata()` proved difficult, so I elected to just interpolate the individual fields here. We don't need `package` for this global pipeline, but this may need to be revisited if we ever add more data to our `_meta` object.